src/simulations/PowerPerturbation.jl

```diff
@@ -14,16 +14,34 @@ struct PowerPerturbation
     node_number
     fraction
     tspan_fault
+    power_symbol
```

Comment:
shouldn't the power_symbol with ":P" be default for a power perturbation? I would not ask for this as an input of the PowerPerturbation struct but make this a default. What do you think?
The if statement in line 30 I would definitely keep to check whether active power is part of the node's field names

Reply:
Good point, i made :P default.

Reply:
great, looks good now!